Update documentation

GlennStrypsteen opened the pull request:
update vegetation documentation

CarolineHalllin committed:
Update README.md

Moved Janelle from developer group to previous contributors

CarolineHalllin committed:
Update model.rst

added vegetation heading

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update and rename bibliography.rst to references.rst

CarolineHalllin committed:
Create introduction.rst

CarolineHalllin committed:
Rename model.rst to model_description.rst

CarolineHalllin committed:
Rename inputfiles.rst to model_setup.rst

CarolineHalllin committed:
Update model_setup.rst

CarolineHalllin committed:
Create tutorials_and_examples.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Create model_interaction.rst

CarolineHalllin committed:
Update model_interaction.rst

CarolineHalllin committed:
Update model_interaction.rst

CarolineHalllin committed:
Update implementation.rst

CarolineHalllin committed:
Update model_interaction.rst

CarolineHalllin committed:
Update introduction.rst

CarolineHalllin committed:
Update model_setup.rst

CarolineHalllin committed:
Update model_setup.rst

CarolineHalllin committed:
Update model_setup.rst

CarolineHalllin committed:
Delete docs/user/output.rst

CarolineHalllin committed:
Update model_setup.rst

CarolineHalllin committed:
Delete docs/user/defaults.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update model_description.rst

CarolineHalllin committed:
Delete docs/user/implementation.rst

CarolineHalllin committed:
Update installation.rst

CarolineHalllin committed:
Add files via upload

CarolineHalllin committed:
Delete docs/user/whatsnew.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update introduction.rst

CarolineHalllin committed:
Update model_description.rst

CarolineHalllin committed:
Update model_description.rst

CarolineHalllin committed:
Update installation.rst

CarolineHalllin committed:
Update index.rst

bartvanwesten committed:
Updated the BMI documentation partially

CarolineHalllin committed:
Update index.rst

bartvanwesten committed:
Merge branch 'update_documentation' of github.com:openearth/aeolis-python into update_documentation

bartvanwesten committed:
Merge branch 'update_documentation' of github.com:openearth/aeolis-python into update_documentation

bartvanwesten committed:
Update the BMI description

bartvanwesten committed:
Update model_interaction.rst

bartvanwesten committed:
Update model_interaction.rst

bartvanwesten committed:
Update model_interaction.rst

CarolineHalllin committed:
Update introduction.rst

CarolineHalllin committed:
Update introduction.rst

CarolineHalllin committed:
Update introduction.rst

christavanijzendoorn committed:
Update constants.py

Beach slope now used in vegetation, runup and erosion calculations, not just the erosion module

CarolineHalllin committed:
Update introduction.rst

christavanijzendoorn committed:
Update model_setup.rst

Adding explanation of process flags

CarolineHalllin committed:
Update introduction.rst

GlennStrypsteen committed:
Update model_description.rst

Update subchapter vegetation

GlennStrypsteen committed:
Add files via upload

Figure vegetation raupach

CarolineHalllin committed:
Add files via upload

GlennStrypsteen committed:
Update model_description.rst

New update vegetation documentation

christavanijzendoorn committed:
Update model_setup.rst

Added process flag descriptions

christavanijzendoorn committed:
Update model_setup.rst

Add threshold flag explanation

GlennStrypsteen committed:
Update aeolis.bib

Added references related to the vegetation documentation

christavanijzendoorn committed:
Update model_setup.rst

Improved layout of process flags

christavanijzendoorn committed:
Update model_setup.rst

Improved layout

CarolineHalllin committed:
updates to index and readme file

CarolineHalllin committed:
Add files via upload

GlennStrypsteen committed:
Update model_description.rst

Updated documentation vegetation and added the references

GlennStrypsteen committed:
Update aeolis.bib

Added some more references related to vegetation documentation

CarolineHalllin committed:
Update aeolis.bib

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update aeolis.bib

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update model_description.rst

CarolineHalllin committed:
Update quickstart.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update model_setup.rst

CarolineHalllin committed:
Add files via upload

CarolineHalllin committed:
Update introduction.rst

CarolineHalllin committed:
Update introduction.rst

GlennStrypsteen committed:
Update model_description.rst

Update vegetation

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Create demonstration_cases

CarolineHalllin committed:
Create shortcourses.rst

CarolineHalllin committed:
Delete docs/tutorials/demonstration_cases

CarolineHalllin committed:
Create demonstration_cases.rst

GlennStrypsteen committed:
Update model_description.rst

GlennStrypsteen committed:
Update model_description.rst

CarolineHalllin committed:
Update index.rst

GlennStrypsteen committed:
Update model_description.rst

GlennStrypsteen committed:
Update model_description.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update README.md

CarolineHalllin committed:
Update model_description.rst

GlennStrypsteen committed:
Update model_description.rst

GlennStrypsteen committed:
Update model_description.rst

GlennStrypsteen committed:
Update introduction.rst

CarolineHalllin committed:
Update demonstration_cases.rst

CarolineHalllin committed:
Update shortcourses.rst

CarolineHalllin committed:
Update demonstration_cases.rst

CarolineHalllin committed:
Update demonstration_cases.rst

CarolineHalllin committed:
Delete docs/user/tutorials_and_examples.rst

CarolineHalllin committed:
Update index.rst

CarolineHalllin committed:
Update demonstration_cases.rst

CarolineHalllin committed:
Update shortcourses.rst

CarolineHalllin committed:
Delete docs/tutorials/2D-parabolic.rst

CarolineHalllin committed:
Delete docs/tutorials/sandmotor.rst

CarolineHalllin committed:
Update demonstration_cases.rst

CarolineHalllin committed:
Update aeolis.bib

CarolineHalllin committed:
Update aeolis.bib

Sierd commented:
The flyout menu on readthedocs is back but in another format. This seems to be a sphinx update. I manually switched some addon button to activate it. 

We can switch between docs versions again.

Sierd committed:
Update aeolis.bib

Strypsteen 2024 was defined twice. This made the references crash. I included an alternative label. The references work again!

Sierd commented:
@GlennStrypsteen, if fixed the bib issue (some of your own publications were defined with using the same name label). Please check if references are still correct. I think one of the references to one of your papers should be adapted as well.

Sierd committed:
dead link removed

christavanijzendoorn committed:
Solve minor layout issues

christavanijzendoorn committed:
Add introductory section to Model Setup

christavanijzendoorn committed:
Solve minor layout issues

christavanijzendoorn committed:
Improve demonstration cases page

christavanijzendoorn committed:
Restructure model description section

christavanijzendoorn committed:
correct small mistake in text model setup

christavanijzendoorn committed:
General tips and getting help

christavanijzendoorn committed:
Update model_setup.rst

fix url

christavanijzendoorn commented:
Hi everyone, I've restructured the model description section and made sure there are no errors anymore in the read the docs build. I've also added some references to the discussion board in the readme and docs. There are several sections in the documentation that still need to be filled in. Please find your assigned tasks below.
- @Sierd:  Model setup > Guidance on solver use & Model description > Numerical implementation of the advection equation. The last one needs to be cleaned up, added to, restructured? Feel free to change whatever is needed.
- @CarolineHalllin Introduction > Model overview & Applicability and limitations, have Sierd review
- @ncohn Model description > Sand fences and Marine erosion.
- @bartvanwesten Model description > Morphological change > Avalanching & wet bed reset
- @GlennStrypsteen Model description > Vegetation, I added subheaders and solved issues with references/math. From what I can see only inundation is not yet explained. 
Based on comments by Jan-Markus and Maddy I think it would also be valuable to add more conceptual figures. So please add visuals if you run into a section that could really use that.

Sierd committed:
Dropdown menu added

Added the dropdown menu
Fixed some bugs

Sierd committed:
cleaned up bib-file, added appendix, explained solvers

Sierd committed:
1D/2D documentation added

Sierd committed:
Merge branch 'main' into update_documentation

bartvanwesten committed:
Quick test

bartvanwesten committed:
Removed test

bartvanwesten committed:
Revise model setup documentation

Updated the model setup documentation to enhance clarity. Example Python scripts are added.

bartvanwesten committed:
Revise model setup documentation 

Updated the model setup documentation for conciseness

bartvanwesten committed:
Included model output documentation with examples

Added detailed explanation of model output, including file format, output variables, and statistical summaries. Included example Python script for visualizing output data.

bartvanwesten committed:
Add example script for animating shear velocity

Added an example Python script for animating shear velocity components using matplotlib.

bartvanwesten committed:
Revise model setup documentation

Updated model setup documentation for conciseness

bartvanwesten committed:
Update documentation for processes, methods, thresholds and boundary conditions

bartvanwesten committed:
Update section title

bartvanwesten committed:
Added animation for guidance on solver schematization.

bartvanwesten committed:
Added guidance on advection and shear solvers

Updated guidance on schematization and numerical solvers in AeoLiS. Enhanced explanations and added case studies for sediment transport simulations.

bartvanwesten committed:
Add sphinxcontrib-video to requirements

bartvanwesten committed:
Add sphinxcontrib.video to Sphinx extensions

bartvanwesten committed:
Update video embedding in model setup documentation

Replaced raw HTML video tags with Sphinx video directive.

bartvanwesten committed:
Fix video paths and clean up HTML in model_setup.rst

bartvanwesten committed:
Update video paths in model_setup documentation

bartvanwesten committed:
Revise schematization guidance and advection solver details

Updated guidance on schematization and advection solver methods, including clarifications and adjustments to case descriptions and results.

bartvanwesten committed:
Update case descriptions and formatting in model setup

bartvanwesten committed:
Add files via upload

bartvanwesten committed:
Add custom CSS to widen content area

bartvanwesten committed:
Configure custom CSS for documentation

Added custom CSS file configuration for documentation.

bartvanwesten committed:
Add custom CSS to widen content area

bartvanwesten committed:
Delete docs/images/_static directory

bartvanwesten committed:
Reduce max-width of content area to 1000px

bartvanwesten committed:
Increase video width from 700 to 900

bartvanwesten committed:
aeolis overview flowchart added

bartvanwesten committed:
Added aeolis overview flowchart to model description

bartvanwesten committed:
Add files via upload

bartvanwesten committed:
Add files via upload

bartvanwesten committed:
Add files via upload

bartvanwesten committed:
Remove figure from model description

Removed figure reference and image from model description.

bartvanwesten committed:
Update model overview figure and description

bartvanwesten committed:
Add files via upload

bartvanwesten committed:
Added video's of landforms to homepage

Removed embedded YouTube video and added video list table.

bartvanwesten committed:
Refactor documentation structure

bartvanwesten committed:
Enhance video grid table styles

Added styles to remove borders and backgrounds from video grid tables, and adjusted padding.

bartvanwesten committed:
Update image display method in introduction.rst

Replaced figure directive with HTML for image display and added image map.

bartvanwesten committed:
Add files via upload

bartvanwesten committed:
Add files via upload

bartvanwesten committed:
Enhance wind shear and topographic steering documentation

Expanded the section on wind shear and topographic steering, detailing the shear stress perturbation method, flow separation, and directional wind modeling techniques in AeoLiS.

bartvanwesten committed:
Update video format from AVI to MP4

bartvanwesten committed:
Add files via upload

bartvanwesten committed:
Delete docs/images/rotating_shear.avi

bartvanwesten committed:
Revise model description for wind shear and flow separation

bartvanwesten committed:
Revise model descriptions and try adding advice

Updated mathematical expressions and added modeling advice for flow parameters.

bartvanwesten committed:
Moved concept topographic steering figure

bartvanwesten committed:
Add files via upload

bartvanwesten committed:
Added draft of short model description

bartvanwesten committed:
Improved short model overview (draft)

bartvanwesten committed:
Add section labels for reference

bartvanwesten committed:
Added more info in quick overview

bartvanwesten committed:
Added reference to quick overview

bartvanwesten committed:
Further edited quick overview section

bartvanwesten committed:
Enhance model setup documentation with grid files section

Added a section on grid files to model setup documentation.

bartvanwesten committed:
Update header levels

bartvanwesten committed:
Fix reference label for time-series section

bartvanwesten committed:
Improved headers and cleaned intro

bartvanwesten committed:
Revise model description

bartvanwesten committed:
Updated model description structure entirely

bartvanwesten committed:
Refactor model description

bartvanwesten committed:
Fix reference for model output section in documentation

bartvanwesten committed:
Clarify solver guide references and subscripts

bartvanwesten committed:
Fix update function to use ustar instead of ustar_mag

bartvanwesten committed:
Trying out different section references

bartvanwesten committed:
Update references and wording in model description

bartvanwesten committed:
Update model description and sediment transport sections

bartvanwesten committed:
Update references and clarify sediment transport section

bartvanwesten committed:
Update model_description.rst

bartvanwesten committed:
Update formatting and section headings in model description

bartvanwesten committed:
Revise model documentation

bartvanwesten committed:
Update references in model_description.rst

bartvanwesten committed:
Add units to various parameters in model description

bartvanwesten committed:
Improved advection scheme

bartvanwesten committed:
Improve section header formatting in model_description.rst

Updated formatting for section headers in model description.

bartvanwesten committed:
Fix formatting in model_description.rst

bartvanwesten committed:
Refactor sediment transport equations in documentation

Updated equations for sediment transport to remove alpha constant and clarify variables.

bartvanwesten committed:
Added saturated transport equations to documentation

bartvanwesten committed:
Added sediment transport velocity documentation

bartvanwesten committed:
Refine lists

bartvanwesten committed:
Enhance model description with additional tips

Added guidance tips for grain speed methods and computational processes.

bartvanwesten committed:
Revise sediment movement note and AeoLiS options

Updated note on sediment movement representation and added explanation of AeoLiS options for momentum balance.

bartvanwesten committed:
Revise vegetation density equation in model description

Updated the equation for vegetation density in the model description.

bartvanwesten committed:
Minor modifications to first part of model description

bartvanwesten committed:
Enhance model description and bed-interaction details

bartvanwesten committed:
Refine model description

bartvanwesten committed:
Fix formula for vegetation density in model description

ssheminway committed:
Comment out Applicability and Limitations section

ssheminway committed:
Remove Applicability and Limitations in introduction.rst

ssheminway committed:
Format section header for applicability and limitations

ssheminway committed:
Update institutions and contact information

ssheminway committed:
Correct capitalization in citation for consistency

ssheminway committed:
Update publication list in introduction

Added new publications for 2024, 2025, and 2026 in the introduction.

ssheminway committed:
Fix typo in model description regarding non-erodible layer

Corrected a typo in tip

ssheminway committed:
Add marine-driven bed level change 

Included brief description and reference to marine driven morphodynamics

jennyhamilton committed:
Update installation.rst

Updated "Installing from source" to match the newer github instructions.

ssheminway committed:
Fix reference and wording in model description

Corrected reference formatting and improved clarity in the marine-driven bed level change section.

ssheminway committed:
Add marine-driven dune erosion description and add formula

Updated terminology and added mathematical formula for wave-driven dune erosion.

ssheminway committed:
Add 'beach evolution' section to 'morphologic change'

Included method_wet_supply and descriptions of methods

ssheminway committed:
'method_wet_supply' figure for documentation 

for 'beach evolution' section in 'morphological change'

ssheminway committed:
Add figure reference for wet supply method

Added figure reference for wet supply method in model description.

ssheminway committed:
Fix formatting in model_description.rst

Correct formatting and spacing issues in the description of the wet_bed_reset method.

jennyhamilton committed:
Update installation.rst

added one space to fix shell

ssheminway committed:
Revise marine-driven dune erosion references and methods

Updated references and terminology related to marine-driven dune erosion and dune erosion processes. Adjusted equations and methods for beach evolution and sediment supply.

jennyhamilton committed:
Update installation.rst

add uv+VS Code installation guide

jennyhamilton committed:
Update installation.rst

made uv installation guide compatible for linux/macOS. cleaned up formatting.

jennyhamilton committed:
Update installation.rst

cleaned up formatting

jennyhamilton committed:
Update installation.rst

fixed formatting

jennyhamilton committed:
Update installation.rst

fixed formatting

jennyhamilton committed:
Update installation.rst

fixing formatting of sections

bartvanwesten committed:
Enhance vegetation dynamics description in AeoLiS

Expanded the vegetation module description in AeoLiS to include details on the new ecomorphodynamic framework and its implications for vegetation dynamics, growth, and mortality. Added specific methods for vegetation density, growth, shear reduction, and interactions with hydrodynamic processes.

bartvanwesten committed:
Add files via upload

bartvanwesten committed:
Enhance vegetation module documentation with figures

Added figures and references to the New Ecomorphodynamic Framework in the vegetation module documentation, enhancing clarity on vegetation growth processes and interactions.

bartvanwesten committed:
Update vegetation module descriptions and methods

bartvanwesten committed:
Enhance vegetation module descriptions in documentation

bartvanwesten committed:
Refine vegetation metrics descriptions and annotations

bartvanwesten committed:
Refactor method descriptions in model documentation

bartvanwesten committed:
Fix references and modify figure dimensions

Updated references and adjusted figure widths in the model description.

bartvanwesten committed:
Improve vegetation method descriptions and metrics

bartvanwesten committed:
Update vegetation metrics and growth method descriptions

bartvanwesten committed:
Revise bed-interaction model description and methods
